priv/repo/migrations/20171113224115_add-a-message-to-the-questionnaire.exs

```diff
@@ -0,0 +1,15 @@
+defmodule :"Elixir.Advisor.Repo.Migrations.Add-a-message-to-the-questionnaire" do
+  use Ecto.Migration
+
+  def up do
+    alter table(:questionnaires) do
+      add :message, :string, null: true
+    end
+  end
+
+  def down do
+    alter table(:questionnaires) do
```

Comment:
Do you really want to delete all the message if you going backwards?

Reply:
Hmm... good call. At the moment I made it so that going forward and coming back leaves the DB in the exact state you started. Keeping the messages would make the `up` tricker, as I'd have to detect if the column already exists...